README.md

```diff
@@ -21,10 +21,10 @@ We are using npm for this particular project (https://www.npmjs.com/get-npm).
 Please check the front end [readme](https://github.com/se701g2/Doto/blob/master/doto-frontend/README.md) for more information on running the front end of the code.
 
 ### Link to the website 
-doto.azurewebsites.net
+[doto.azurewebsites.net](doto.azurewebsites.net)
```

Comment:
Wow that was fast. I think the link requires http:// prefix for it to work. It currently links to a github page.

Reply:
Yeah, I just added it